lib/matplotlib/mlab.py

```diff
@@ -3824,8 +3817,8 @@ def poly_below(xmin, xs, ys):
       xv, yv = poly_below(0, x, y)
       ax.fill(xv, yv)
     """
-    if ma.isMaskedArray(xs) or ma.isMaskedArray(ys):
-        numpy = ma
+    if isinstance(xs, np.ma.MaskedArray) or isinstance(ys, np.ma.MaskedArray):
+        numpy = np.ma
```

Comment:
This (previous and present) use of numpy for np.ma or ma looks odd.  Maybe switch it to `nx` or `num`?  Long ago, in the days when we had to support Numeric, numarray, and numpy, we used `nx` as the shim module.